client/src/app/pages/sbom-list/sbom-list.tsx

```diff
@@ -322,8 +337,11 @@ export const SbomExpandedArea: React.FC<SbomExpandedAreaProps> = ({
         <Thead>
           <Tr>
             <TableHeaderContentWithControls {...tableControls}>
-              <Th {...getThProps({ columnKey: "name" })} />
-              <Th {...getThProps({ columnKey: "version" })} />
+              <Th aria-label="name" {...getThProps({ columnKey: "name" })} />
+              <Th
+                aria-label="version"
+                {...getThProps({ columnKey: "version" })}
+              />
```

Comment:
same here, there should be no need to add aria-label. 
If we start adding `aria-label` to one, then we might need to add it everywhere and although looks lazy I think we could avoid adding things and keep it simple unless required